Specs/Number/Italian/NumberModel.json

```diff
@@ -16,72 +185,85 @@
   {
     "Input": "192.168.1.2",
     "NotSupported": "dotnet",
```

Comment:
Same comments as before. Why is this disabled in dotnet?
And please try not to re-order specs. Keep the order as similar to the original one in English. No need to re-order now, but keep it in mind on next PRs.

Reply:
This case was disabled because it fails proper exaction as is (From English Counterpart). 
It is probably due to DecimalSeparatorChar differences, other similar languages have the same Spanish and French. In these two languages the test case is presented differently, in French as: 
  {
    "Input": "192.168.1.2",
    "Results": [
      {
        "Text": "192.168",
        "TypeName": "number",
        "Resolution": {
          "value": "192168"
        },
        "Start": 0,
        "End": 6
      },
      {
        "Text": "1",
        "TypeName": "number",
        "Resolution": {
          "value": "1"
        },
        "Start": 8,
        "End": 8
      },
      {
        "Text": "2",
        "TypeName": "number",
        "Resolution": {
          "value": "2"
        },
        "Start": 10,
        "End": 10
      }
    ]
  }

Reply:
Can you update the test to follow French? You can also add a "Comment" tag in the test case to explain the reason for the change, so its properly documented for future devs.

Reply:
yes, we will do that.